apps/open-swe/src/graphs/planner/nodes/proposed-plan.ts

```diff
@@ -151,12 +159,37 @@ export async function interruptProposedPlan(
     runInput.taskPlan,
   );
 
+  // Create custom node event for accepted plan
+  const acceptedPlanEvent: CustomNodeEvent = {
+    nodeId: ACCEPTED_PLAN_NODE_ID,
+    actionId: uuidv4(),
+    action: "Plan accepted",
+    createdAt: new Date().toISOString(),
+    data: {
+      status: "success",
+      planTitle: state.proposedPlanTitle,
+      planItems: planItems,
+      interruptType: interruptRes.type,
+    },
+  };
+
+  // Create AIMessage with accepted plan data
```

Comment:
unnecessary comments (I need to prompt open swe to do a better job at not leaving these, but until then we should manually delete)
```suggestion
  const acceptedPlanEvent: CustomNodeEvent = {
    nodeId: ACCEPTED_PLAN_NODE_ID,
    actionId: uuidv4(),
    action: "Plan accepted",
    createdAt: new Date().toISOString(),
    data: {
      status: "success",
      planTitle: state.proposedPlanTitle,
      planItems: planItems,
      interruptType: interruptRes.type,
    },
  };
```